python/ray/autoscaler/kubernetes/example-full.yaml

```diff
@@ -3,7 +3,7 @@ cluster_name: example-cluster
 
 # The maximum number of workers nodes to launch in addition to the head
 # node.
-max_workers: 2
+max_workers: 3
```

Comment:
did you accidentally push this? why max_workers=3?

Reply:
fixed